Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -26,42 +26,64 @@ public partial class TapToPlace : MonoBehaviour
         /// </summary>
         WorldAnchorManager anchorManager;
```

Comment:
I think I pushed changes with the public/private stuff.  What's the norm, should I respond to each individually?